openstack/python/tar/bin/openstack-describe-instances

```diff
@@ -32,7 +32,8 @@ class OpenStackDescribeInstances(DescribeInstancesCommand, OpenStackCommand):
                  2: 'Terminated',
                  3: 'Pending',
                  4: 'Unknown',
-                 5: 'Stopped'}
+                 5: 'Stopped',
+                 7: 'Error'}
```

Comment:
It looks like `latest` point to [`1.0.0-pre1`](https://libcloud.readthedocs.org/en/latest/_modules/libcloud.html). The source code for `1.0.0-pre1` is probably not based on the latest `0.x` version which is very bad.
So everybody who check the documentation or the source code for `latest` (which is the default) are looking at a very old version instead of the very new one.
This is very very bad and also good to know.